ch1/create_table.py

```diff
@@ -1,12 +1,12 @@
+# create_table.py
 import os
 
 import psycopg2
-from psycopg2.extensions import connection
 
 DB_HOST = os.getenv("DB_HOST", "localhost")
 
 
-def create_table(db_connect: connection) -> None:
+def create_table(db_connect) -> None:
```

Comment:
```suggestion
def create_table(db_connect):
```
Return 도 없애버리면 어떨까요?